### ISSUE TITLE
Some Tasks 

### ISSUE THREAD

**Issue by kimiamavon** [HeyItsRiddhi/DSFinalProject#3]
What do you guys think about this to help us divide and conquer? 

Riddhi - Code Review, EDA, start making web layout for final? (see Nick's repo from last year)
@HeyItsRiddhi 

William- continue working on EDAs and teasing out trends (but consistent style of EDAs). Help me get EDAs to support the models. 

Humza - work on ethnicity classifier. Let me know if you need help with anything.
@humzatahir 

Kimia - code review, work on building a few different models. Determine what works best @ what params? Once these initial models our built, we'll run the same algo using the ethnicity classifier.

What do you guys think? If I am giving you too much let me know, I'm open to feedback

**Comment by HeyItsRiddhi** [comment]
Perfect!!!!

Sent from my iPhone

On Nov 30, 2017, at 8:57 AM, kimiamavon <notifications@github.com<mailto:notifications@github.com>> wrote:


What do you guys think about this to help us divide and conquer?

Riddhi - Code Review, EDA, start making web layout for final? (see Nick's repo from last year)
@HeyItsRiddhi<https://github.com/heyitsriddhi>

William- continue working on EDAs and teasing out trends (but consistent style of EDAs). Help me get EDAs to support the models.

Humza - work on ethnicity classifier. Let me know if you need help with anything.
@humzatahir<https://github.com/humzatahir>

Kimia - code review, work on building a few different models. Determine what works best @ what params? Once these initial models our built, we'll run the same algo using the ethnicity classifier.

What do you guys think? If I am giving you too much let me know, I'm open to feedback

**Comment by kimiamavon** [comment]
**In milestone 3:** 

Florida- show diff voting behaviers (diff ethnicities per county, diff than US, REP/DEM split between counties). Is it representative of the US or is it very different? CAN WE SHOW HOW ITS DIFF? 

2nd sent drop- clearly define predictor and response (age, location, we are going to design a costume class to claff ethnicity from name to assist in this model). 


Discuss all predictors together, baseline class... AXIS SHOULD BE CLEAR...Comment, notice this is out of x but we scaled it to make it so you can see. 

Predict ethnicity based on name. We should have quoted the accuracy. mention other dataset accuracy. Work on how we tell the story. Make it clear what data is validation, training data. "PREDICTED ETHNICITY" 


Could we do a map of Florida. Mapping in tableau.  @kmavon 
Where voters are turning out. 
- %% voter turnout 
- Excluding white, the majority ethnicity. 
- heatgraph hispanic (dark to light)

Baseline model is ridge, lesso with CV regular sore via 5 fold xvalid - @kmavon 

Accuracy was X n the test set. Better than the other one. 


Story about model improvement: 


Modeling- present baseline, no ethnicity. BE CLEAR, predicting w.o ethnicity, using these predictors. 

We need a classification metric to put in the metric. 
3.1 Baseline
3.2 ethnicity metric (accuracy, how good is it). Wanting to drop out those below the threshhold. Good prediction on 10% of extreme names. Ones in between will add noise to the model. High precise low recall. 
3.3. Tie first two together to make a better model. High level model. Using predictors ot make a better score. 0.25 v .21. 

section 1- where data from?? 250 is INSTANCES of Voting @william96 
Section 2- EDA and charts, voter turnout, here are pretty maps @HeyItsRiddhi  @william96 
Section 3- @humzatahir @kmavon build models
Section 3.99999 --> @humzatahir @william96 @kmavon 
work together. is .25 better? Subsample test set, is it better? We have a mean accuracy score....we could run a formal hypothesis test. Bootstrap over scores. 95% percentile doesnt overlap, confident model is doing. T test (list of scores, std dev).... due to the subsample of data, we are statistically confident that .25 is better than .21. 
(bootstrap: number of smaller test sets, histogram of score. CV the test. Is our score genuinely better). 
what is the standard error? Given the fact that St Error is around score estimate. 

What do the residuals look like? 

VALIDE ASSUMPTIONS @humzatahir @william96 @kmavon 
Define all the models we chose. "Lasso regression....these assumptions... these assumptions hold/dont for these reasons (robust etc)." 

Almost ensamble model, using what we can get to make it more robust. Did it do what we expect to do?
Where did name and ethnicity classifier come from, this is why its representative. What is the coverage of names we used @humzatahir 
Ethnicity data give a good prediction.

**Comment by kimiamavon** [comment]
make a table: 
Baseline classifier (3.1) ======= .21
+ Ethnicity (3.2) (predicted) (3.3)= .25
